doc/dev/binary-file-formats.rst

```diff
@@ -56,6 +56,7 @@ Use Bigcount        1       6   ``0x01`` if bigcounts is used, else ``0x00``
 K-size              4       7   k-mer length, ``ht._ksize``. [``uint32_t``]
 Number of Tables    1      11   Number of Count-min Sketch tables,
                                 ``ht._n_tables``. [``uint8_t``]
+Occupied Bins       8      12   Number of occupied bins
```

Comment:
There's an issue on this

On Wed, Aug 12, 2015 at 3:09 PM Jake Fenton <notifications@github.com>
wrote:

> In doc/dev/binary-file-formats.rst
> <https://github.com/dib-lab/khmer/pull/1247#discussion_r36920570>:
>
> > @@ -56,6 +56,7 @@ Use Bigcount        1       6   ``0x01`` if bigcounts is used, else ``0x00``
> >  K-size              4       7   k-mer length, ``ht._ksize``. [``uint32_t``]
> >  Number of Tables    1      11   Number of Count-min Sketch tables,
> >                                  ``ht._n_tables``. [``uint8_t``]
> > +Occupied Bins       8      12   Number of occupied bins
>
> What are the preferred extensions?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dib-lab/khmer/pull/1247/files#r36920570>.
>
-- 
Michael R. Crusoe: Programmer & Bioinformatician crusoe@ucdavis.edu
The lab for Data Intensive Biology; University of California, Davis
https://impactstory.org/MichaelRCrusoe http://twitter.com/biocrusoe